integration-tests/utils/integration-helpers.js

```diff
@@ -58,7 +58,7 @@ const createEntityInWorkspace = (page, billingProject, workspaceName, testEntity
 const makeUser = async () => {
   const { email } = await fetchLyle('create')
   const { accessToken: token } = await fetchLyle('token', email)
-
+  console.info(`created a user with token: ...${token.toString().substr(-10, 10)}`)
```

Comment:
is it worth declaring a helper somewhere above?

```js
const clipToken = str => str.toString().substr(-10, 10)
```

Reply:
I like that idea.
Also, nit: I think `substr(-10)` is sufficient.